avr/cores/megacommand/MCL/A4Track.cpp

```diff
@@ -5,14 +5,17 @@
 
 void A4Track::load_seq_data(int tracknumber) {
 #ifdef EXT_TRACKS
-  if (seq_data.speed == 0) { seq_data.speed = EXT_SPEED_2X; }
```

Comment:
how about distinguishing old data format with EXT_TRACK_TYPE and A4_TRACK_TYPE? allocate new track types for new format.

Reply:
this could be more generic as it doesn't concern the details of the data being held.

Reply:
we could do it this way as well. the idea was to add versioning variable once this initial conversion is done

Reply:
this will work if we want to increase the number of locks for MD track, without thinking of another (quite arbitrary) version probe for the MD track.

Reply:
> we could do it this way as well. the idea was to add versioning variable once this initial conversion is done

Yeah I think the two can co-exist. The idea is that the finer versioning insider a container is a track semantic, while the row header describes the container itself.